entities/buildinfo.go

```diff
@@ -343,6 +343,7 @@ type Module struct {
 	Artifacts         []Artifact   `json:"artifacts,omitempty"`
 	ExcludedArtifacts []Artifact   `json:"excludedArtifacts,omitempty"`
 	Dependencies      []Dependency `json:"dependencies,omitempty"`
+	Parent            string       `json:"parent,omitempty"`
```

Comment:
Let's add a clear comment for this new field, explaining its content and purpose.